### PR TITLE
FIXED File Format and Button Responsiveness

### PR DESCRIPTION
**Pull request by syazah**

### CODE REVIEW

File: index.js
Comment (sambhavsaxena) [inline index.js:1]: you can eliminate this check by setting the upload type to image/png/jpeg.

Comment (syazah) [reply index.js:1]: Yes it can be done through tailwind but to make an alert I used js

Comment (sambhavsaxena) [reply index.js:1]: [here](https://stackoverflow.com/questions/3828554/how-to-allow-input-type-file-to-accept-only-image-files)'s how you do it in a very straightforward manner.

Comment (syazah) [reply index.js:1]: Thank you for pointing out, but just to give an alert I used the former method

**Pull request merged**